docs/landing/data/releases.toml

```diff
@@ -1,12 +1,12 @@
-current = "4.1.1"
+current = "4.2.0"
```

Comment:
Is there a way to reuse this value so that we only need to update it in a single place? If it cannot be reused directly from `releases.toml` it probably can be specified in `config.toml` as a [site parameter](https://gohugo.io/variables/site/#the-siteparams-variable) and then use the parameter value in pages.

Reply:
Perhaps, but can we investigate that separately?